_utility/send_gh_comment.sh

```diff
@@ -1,34 +1,43 @@
 #!/bin/bash
-set -e
+set +e
```

Comment:
That is done in "send_gh_comment_validation.sh" but we splitted this in 2 scripts due to its verbosity 👍